src/pages/docs/en/basics/local.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Developing Locally
+section: Basics
+position: 3
+hidden: false
+layout: "@docs"
+---
+
+## Starting a Development Server
+
+Once you've created a Space project, you can start developing your app locally using the Space CLI's `space dev` command.
+
+```bash
+space dev # start a local development server and open the app in your browser
+```
+
+Behind the scenes, the Space CLI is:
+
+- Generating a data key to allow your app to access the Bases and Drives of your project
+- Running each of your micro in a separate process
+- Exposing all of your micros through a single entrypoint, on `localhost:4200` by default.
+
+If you want to open the app in your browser, you can use add `--open` flag:
+
+```bash
+space dev --open # start a local development server and open the app in your browser
+```
+
+You can customize the development command for each micro in your `Spacefile` using the `dev` key.
+
+```yaml
+v: 0
+micros:
+  - name: client
+    src: ./client
+    engine: svelte
+    primary: true
+    path: client
+  - name: api
+    src: ./api
+    engine: nodejs16
+    dev: nodemon index.js # start the micro in development mode
+    path: api
+```
+
+The specified `dev` commands need to start a web server that listens on the port specified with the `PORT` environment variable. This way the CLI can properly forward requests to the micro.
+
+You can find guide specific to each micro engine in the Quickstart Guides.
+
+## Advanced Usage
+
+If you want to have more control over the development process, you can also split the `dev` command into two parts:
+
+- First, start micros individually using the `space dev up` command
+- Then, start the entrypoint server using the `space dev proxy`
```

Comment:
"entrypoint server" sounds a bit weird lol. Don't know a better name though except proxy

Reply:
yeah proxy is fine, it is in the command name